### ISSUE TITLE
Dealing with vote explanations

### ISSUE THREAD

**Issue by joepio** [popolo-project/popolo-spec#123]
In Dutch parliament, both individuals and parties can give an explanation to why they voted in a certain way. 

In issue #57, @jpmckinney coined a possible 'note' attribute for votes. I feel like it would be better to have a more descriptive 'VoteExplanation' name. ~~I also think that it should be its own resource, since it can be linked to both votes by individuals and votes by groups (counts).~~

But perhaps there is a better way of dealing with these explanations. What are your thoughts on this?

**Comment by jpmckinney** [comment]
@joepsz I'm not clear on why the explanation needs to be its own object. Can you explain? [Vote](http://www.popoloproject.com/specs/vote.html) already allows votes by individuals or groups. If we want to add a note to the `Count` class, that's easy, too.

**Comment by joepio** [comment]
You are right, the property / attribute could be shared between Vote and Counts. 

However, I'm a bit confused about using Vote for groups (specifically political parties). It creates problems when doing calculations on votes, since the vote of a party has a certain weight. Because of this, we use a Count for each political party (and each option, if its members voted differently). And if we do that, we still need a VoteExplanation (or Note) property on Counts as well as Votes.
